mzLib.nuspec

```diff
@@ -11,51 +11,34 @@
-    <file src="Proteomics\bin\Release\netstandard2.0\Proteomics.dll" target="lib\netstandard2.0" /> 
-	<file src="BayesianEstimation\bin\Release\netstandard2.0\BayesianEstimation.dll" target="lib\netstandard2.0" /> 
-    <file src="Proteomics\bin\Release\netstandard2.0\ProteolyticDigestion\proteases.tsv" target="lib\netstandard2.0\ProteolyticDigestion" /> 
-    <file src="UsefulProteomicsDatabases\bin\Release\netstandard2.0\UsefulProteomicsDatabases.dll" target="lib\netstandard2.0" /> 
-	<file src="ThermoRawFileReader\bin\Release\netstandard2.0\ThermoFisher.CommonCore.BackgroundSubtraction.dll" target="lib\netstandard2.0" /> 
-	<file src="ThermoRawFileReader\bin\Release\netstandard2.0\ThermoFisher.CommonCore.Data.dll" target="lib\netstandard2.0" /> 
-	<file src="ThermoRawFileReader\bin\Release\netstandard2.0\ThermoFisher.CommonCore.MassPrecisionEstimator.dll" target="lib\netstandard2.0" /> 
-	<file src="ThermoRawFileReader\bin\Release\netstandard2.0\ThermoFisher.CommonCore.RawFileReader.dll" target="lib\netstandard2.0" /> 
-	<file src="ThermoRawFileReader\bin\Release\netstandard2.0\ThermoRawFileReader.dll" target="lib\netstandard2.0" /> 
-
-    <file src="Chemistry\bin\Release\net471\Chemistry.dll" target="lib\net471" />
-    <file src="FlashLFQ\bin\Release\net471\FlashLFQ.dll" target="lib\net471" />
-    <file src="MzIdentML\bin\Release\net471\MzIdentML.dll" target="lib\net471" />
-    <file src="MzLibUtil\bin\Release\net471\MzLibUtil.dll" target="lib\net471" />
-    <file src="MzML\bin\Release\net471\MzML.dll" target="lib\net471" /> 
-    <file src="Mgf\bin\Release\net471\Mgf.dll" target="lib\net471" /> 
-    <file src="PepXML\bin\Release\net471\PepXML.dll" target="lib\net471" /> 
-    <file src="MassSpectrometry\bin\Release\net471\MassSpectrometry.dll" target="lib\net471" /> 
-    <file src="Proteomics\bin\Release\net471\Proteomics.dll" target="lib\net471" /> 
-	<file src="BayesianEstimation\bin\Release\net471\BayesianEstimation.dll" target="lib\net471" />
-    <file src="Proteomics\bin\Release\net471\ProteolyticDigestion\proteases.tsv" target="lib\net471\ProteolyticDigestion" /> 
-    <file src="UsefulProteomicsDatabases\bin\Release\net471\UsefulProteomicsDatabases.dll" target="lib\net471" /> 
-	<file src="ThermoRawFileReader\bin\Release\net471\ThermoFisher.CommonCore.BackgroundSubtraction.dll" target="lib\net471" /> 
-	<file src="ThermoRawFileReader\bin\Release\net471\ThermoFisher.CommonCore.Data.dll" target="lib\net471" /> 
-	<file src="ThermoRawFileReader\bin\Release\net471\ThermoFisher.CommonCore.MassPrecisionEstimator.dll" target="lib\net471" /> 
-	<file src="ThermoRawFileReader\bin\Release\net471\ThermoFisher.CommonCore.RawFileReader.dll" target="lib\net471" /> 
-	<file src="ThermoRawFileReader\bin\Release\net471\ThermoRawFileReader.dll" target="lib\net471" /> 
-
-    <file src="mzLib.targets" target="build\net471" />
+    <file src="BayesianEstimation\bin\Release\netcoreapp3.0\BayesianEstimation.dll" target="lib\netcoreapp3.0" />
```

Comment:
Did you test packaging the nupkg and then using it in MetaMorpheus to check that it works?

Reply:
yep! I had to make a .NET core 3 metamorpheus build, but all is OK. I'd like to test more on a Linux machine but that's not necessarily required to merge the PR. I just want to test getting computer diagnostic info on different operating systems.